toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -361,3 +367,27 @@ func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safec
 
 	return nil
 }
+
+func addCustomizerRelease(imageChroot *safechroot.Chroot) error {
+	var err error
+
+	logger.Log.Infof("Creating image customizer release file")
+
+	// TODO: update micVersion once the mic release process is in place
+	micVersion := "0.0.0"
+	currentTime := time.Now().Format("2006-01-02T15:04:05Z")
```

Comment:
It might be goodness to retrieve the date/time at the start of the customization process and pass it down to the functions that need it. That way, if another component needs this value, then the values won't disagree because `time.Now()` was called by multiple functions.

It will also make it easier to write the UT (as per my other comment) since you can pass a fixed time value into this function.

Reply:
moved time.Now() to the beginning of the doCustomizations function, let me know if it doesnt make sense